src/services/api/AuthService.ts

```diff
@@ -87,7 +74,15 @@ class AuthService extends ApiService {
   login = async (
     credentials: CredentialsLogin
   ): Promise<UserCredentialsProp> => {
+    if (!credentials['username']) {
+      // the login page doesnt have the username input field
+      credentials['username'] = credentials['email'] // in this case the BE is expecting the username to be the email
+    }
```

Comment:
Please avoid mutation of passed props...
You should use something like this instead.
```TS
const { data } = await this.apiClient.post(ENDPOINTS.LOGIN, { ...credentials, username: credentials.email })
```